content/docs/standard-library/io/_index.md

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
Please follow stdlib structure. All these things are entities of the same package `builtin`. When I said "separate file per package" I meant package in terms of Nevalang package.